doc/src/find_and_immediate.md

```diff
@@ -158,3 +158,18 @@ If the user's architecture is not listed above they will need to run the Find AP
 ### Backend Limitations
 
 OpenCL support for immediate mode via the fallback is limited to fp32 datatypes. This is because this current release's fallback path goes through GEMM which on the OpenCL is serviced through MIOpenGEMM -- which itself only contains support for fp32. The HIP backend uses rocBLAS as its fallback path which contains a richer set of datatypes.
+
+
+### Find Modes
+
+MIOpen provides a set of Find modes which are used to accelerate the Find calls. The different modes are set by using the environment variable `MIOPEN_FIND_MODE`, and setting it to one of the values:
```

Comment:
Since MIOpen's API doesnt support changing the find mode, this section should go under the debugging section.

Reply:
That is true. I'll link to that from here as well.

Reply:
Please note that this is not a debugging functionality, but performance optimization, and it affects the behavior of Find API.